tests/e2e/manifest/target-elastiservice.yaml

```
@@ -0,0 +1,19 @@
+metadata:
+  name: target-elastiservice
+spec:
+  cooldownPeriod: 30
+  minTargetReplicas: 2
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: deployments
+    name: target-deployment
+  service: target-deployment
+  triggers:
+  - metadata:
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)